### PR TITLE
prepare `v0.5`

### PR DESCRIPTION
**Pull request by Rykath**
* [x] add *Getting Started* page to documentation
  * explain the necessary steps to set up proFit with a simulation
  * reference further reading: variables, surrogates, cluster support, ...
* [x] add version information to UI
* [x] add `--debug` flag to UI to explicitly enable dash debug mode
* [x] fix import paths for dash
  * Chris: verified that it works for python 3.7
* [x] add & update zenodo metadata
  * [x] add the first three paragraphs of the README to zenodo, & the list of features
* [x] update keywords
* [x] default help message

<details><summary>preview of Zenodo description</summary>

image
</details>

<details><summary>preview of Getting Started docs</summary>

image
image
image
</details>

<details><summary>display help message on argument error</summary>

```
$ profit
error: the following arguments are required: mode
usage: profit [-h] mode ... [base-dir]

Probabilistic Response Model Fitting with Interactive Tools v0.5.dev58+gbd90fab

positional arguments:
  mode
    run       start simulation runs
    fit       fit data (e.g. with a Gaussian Process)
    ui        interactively visualize results using dash
    clean     remove temporary files, run directories and logs
    version   show version information
  base-dir    path to config file or directory containing profit.yaml (default: current working
              directory)

options:
  -h, --help  show this help message and exit
```
</details>

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -12,7 +12,7 @@ url = https://github.com/redmod-team/profit
 project_urls = 
     Issue Tracker = https://github.com/redmod-team/profit/issues
     Documentation = https://profit.readthedocs.io/en/latest
-keywords = PCE, UQ
+keywords = Parameter Study, GP, HPC, Active Learning
```
Comment (Rykath) [inline setup.cfg:15]:
```suggestion
keywords = Parameter Study, Gaussian Process, HPC, Active Learning
```

File: .zenodo.json [hunk 2]
```diff
@@ -0,0 +1,47 @@
+{
+   "description":"Collection of tools for studying parametric dependencies of black-box simulation codes or experiments and construction of reduced order response models over input parameter space.",
+   "license":"MIT",
+   "title":"proFit: Probabilistic Response Model Fitting with Interactive Tools",
+   "creators":[
```
Comment (Rykath) [inline .zenodo.json:5]: -> alphabetic

File: .zenodo.json [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+      {
+         "orcid":"0000-0003-4773-416X",
+         "affiliation":"Technische Universit\u00e4t Graz",
+         "name":"Christopher Albert"
+      },
+      {
+         "affiliation":"Technische Universit\u00e4t Graz",
+         "name":"Maximilian Kendler"
+      },
+      {
```
Comment (Rykath) [inline .zenodo.json:15]: → alphabetic
add Richard Hofmeister

**Review by krystophny — COMMENTED** [review]
Looks fine, thanks!

**Pull request merged**